molo/core/tests/test_views.py

```diff
@@ -1656,7 +1651,7 @@ def test_article_recommended_section_untranslated(self):
         response = self.client.get(
             '/sections-main-1/section-a/article-a-in-french/')
         self.assertEquals(response.status_code, 200)
-        self.assertContains(response, self.article_b.title)
+        self.assertContains(response, self.article_a.title)
```

Comment:
I don't think this change is correct?
The check was to ensure that article_b is visible in the recommended section when you're looking at article_a.